.github/workflows/tests_lgpumpi_python.yml

```diff
@@ -112,10 +112,6 @@ jobs:
           python -m pip install custatevec-cu${{ matrix.cuda_version_maj }} mpi4py openfermionpyscf
           PL_BACKEND=lightning_qubit python -m pip install . -vv
 
-      - name: Downgrade Scipy (remove after release)
-        if: inputs.lightning-version == 'stable'
-        run: pip install --force-reinstall "scipy<1.14"
-
```

Comment:
Great! Thanks!

Reply:
Thanks for fixing this in the first place.

Reply:
Is this related to the changes in symbol names brought in scipy-1.14?